core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -43,6 +47,9 @@ sealed trait TLVParentFactory[T <: TLV] extends Factory[T] {
 
 object TLV extends TLVParentFactory[TLV] {
 
+  val FALSE_BYTE: Byte = 0x00
```

Comment:
Isn't this a `IS_UNSIGNED` and `IS_SIGNED`  or something like this?

Reply:
Boolean can be used for other TLVs